readme.md

```diff
@@ -163,6 +163,30 @@ Once the application has been created via `eb create ...` all subsequent updates
 eb deploy ${environment name} --profile ${your AWS profile}
 ```
 
+#### Switching Loadbalancer settings between accounts
+
+Loadbalancer settings is required for HTTPS to run behind the load balancers for Elastic Beanstalk. Each Loadbalancer setting is unique to each AWS profile, so switching configuration files is required to deploy to different AWS profiles.
+
+| AWS Profile | Configuration File | 
+|---|---|
+| `nypl-digital-dev` | `https-nypl-digital-dev.config` | 
+| `nypl-sandbox` | `https-nypl-sandbox.config` | 
+
+When `development` is ready to merge into `qa`, only include `https-nypl-digital-dev.config` within `.ebextensions` directory. Since CI/CD is being performed via Travis CI, the following needs to be done on the local `qa` branch, after a merge with the most up-to-date local `development` branch, before pushing into remote `qa`:
+
+```bash
+git checkout qa
+git merge development
+git rm .ebextensions/https-nypl-sandbox.config
+git add .ebextensions/https-nypl-digital-dev.config
+git commit
+git push origin qa
+```
+
+At `git push origin qa`, Travis CI will be triggered and will deploy to QA server with the correct credentials.
```

Comment:
Travis CI will deploy to the QA server with the correct credentials.

Shouldn't we mention production as well in this section regarding the appropriate credentials being deployed to the correct environment?

Reply:
I think we should.